### ISSUE TITLE
[sentry-unplugin] Update Source Maps Docs for newly supported JS Bundler

### ISSUE THREAD

**Issue by smeubank** [getsentry/sentry-docs#5718]
### Core or SDK?

Platform/SDK

### Which part? Which one?

sentry-javascript

### Description

JavaScript SDK team is working on a new feature which will support 3 new JS bundlers in the uploading of Source Maps to Sentry. ideally with same feature parity of what is supported through webpack today.

More information can be found here: https://github.com/getsentry/sentry-javascript-bundler-plugins

Today via sentry-webpack-plugin we only support webpack, and for other JS Bundlers we direct people down an advnaced usage path, by setting everything up manually with sentry-cli.

### Suggested Solution

Docs shall be updated to include guides for Vite, esbuild, and Rollup

Docs to be updated: (multiple issues can be created from this one)

**Comment by imatwawana** [comment]
@Lms24 - I'm concerned about making the wizard pages significantly longer. How much more content would that entail? All of the content on the upload pages that I'm seeing on #5799? Or less? Or more?

**Comment by Lms24** [comment]
@imatwawana good point. This probably depends a little on the JS framework. Ideally, I think it would be just one more step, showing how to set up bundler plugins. This however doesn't work in all frameworks (e.g. Angular, Ember?, Remix?). So for the frameworks where it doesn't work, I guess our best strategy would be to link to Sentry-CLI/Release action instructions. Which I'm realizing is far from ideal.

**Comment by imatwawana** [comment]
What about only doing it for the bundler plugins and if there isn't an ideal strategy, not adding the step?
